build.gradle

```diff
@@ -33,14 +33,10 @@ ext {
     VERSION_CODE = 11
     MIN_SDK_VERSION = 14
     // If you change the SDK version don't forgot to update .travis.yml
-    TARGET_SDK_VERSION = 26
-    COMPILE_SDK_VERSION = 26
-
-    supportLibraryVersion = '26.0.0'
-    // compositeAndroidVersion should be always the same version as the supportLib version
-    // Since composite is not up to date we can't use the latest one
-    compositeAndroidVersion = supportLibraryVersion
+    TARGET_SDK_VERSION = 28
+    COMPILE_SDK_VERSION = 28
 
+    androidXLibraryVersion = '1.0.0'
```

Comment:
now with semantic versioning, we can't use the same version for all deps.

Reply:
As of today "nothing" have changed.
We will see if Google will *really* follow this pattern.
As time of writing this is totally acceptable..